src/data/weapon_multipliers.ts

```diff
@@ -674,6 +674,34 @@ const DPS_WEAPON_MULTIPLIERS = Object.freeze({
         value: 30,
       }),
     ],
+    "Sagittarius Astra": [
```

Comment:
I know absolutely nothing about DA but I noticed none of these are set to active = true, is that intended?

Reply:
Fixed